lokilogs/lokilog.go

```diff
@@ -0,0 +1,86 @@
+package lokilogs
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"time"
+
+	objConsulAgent "github.com/carwale/golibraries/consulagent"
+	"github.com/carwale/golibraries/gologger"
+)
+
+var (
+	globalConsulAgent *objConsulAgent.ConsulAgent
+	isLokiLogEnabled  bool
+	serviceLogger     *gologger.CustomLogger
+	globalserviceName string
+)
+
+// SetBasicConfig start point of the request
+func SetBasicConfig(key string, consulIP string, logger *gologger.CustomLogger, serviceName string) {
+	globalConsulAgent = objConsulAgent.NewConsulAgent(
+		objConsulAgent.ConsulHost(consulIP),
+		objConsulAgent.Logger(logger),
+	)
+	serviceLogger = logger
+	globalserviceName = serviceName
+
+	go checkLokiLogStatus(key)
+}
+
+func checkLokiLogStatus(key string) {
+	for {
+		time.Sleep(10 * time.Second)
```

Comment:
this time was added for testing purpose. Will update it to 15minutes once branch is approved